packages/test/test-end-to-end-tests/src/test/summarizerWithLocalChanges.spec.ts

```diff
@@ -85,20 +93,54 @@ const dataStoreFactory1 = new DataObjectFactory(
 	[],
 	[],
 	[],
-	mixinSummaryHandler(getComponent),
+	mixinSummaryHandler(getDataObject),
 );
 
 const registryStoreEntries = new Map<string, Promise<IFluidDataStoreFactory>>([
 	[rootDataObjectFactory.type, Promise.resolve(rootDataObjectFactory)],
 	[dataStoreFactory1.type, Promise.resolve(dataStoreFactory1)],
 ]);
-const runtimeFactory = new ContainerRuntimeFactoryWithDefaultDataStore(
-	rootDataObjectFactory,
-	registryStoreEntries,
-	undefined,
-	[],
-	runtimeOptions,
-);
+
+const settings = {};
+settings["Fluid.ContainerRuntime.Test.SummaryStateUpdateMethod"] = "restart";
+settings["Fluid.ContainerRuntime.Test.CloseSummarizerDelayOverrideMs"] = 0;
+
+const createContainer = async (
+	provider: ITestObjectProvider,
+	disableSummary: boolean = true,
+	logger?: ITelemetryLogger,
+): Promise<IContainer> => {
+	let summaryConfigOverrides: ISummaryConfiguration;
+	if (disableSummary) {
+		summaryConfigOverrides = { state: "disabled" };
+	} else {
+		const IdleDetectionTime = 10;
+		summaryConfigOverrides = {
+			...DefaultSummaryConfiguration,
+			...{
```

Comment:
why not just inline these 3?  the `...{}` is a no-op right?

Reply:
I get the following error on doing that:
![image](https://github.com/microsoft/FluidFramework/assets/45832642/2585422d-a7f5-4b67-a706-1aa8d18a1ab4)

Reply:
gotcha